src/main/java/com/example/controllers/userControllers/UserController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.controllers.userControllers;
+
+import com.example.dto.UserDto;
+import com.example.entity.User;
+import com.example.services.UserService;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import javax.annotation.Resource;
+
```

Comment:
JD

Reply:
@Vlada0612 Following your advice the java dock was added.